.circleci/config.yml

```diff
@@ -89,8 +92,7 @@ commands:
     steps:
       - install-gems
       - run:
-          name: Update git commit in SPM package
-          working_directory: IntegrationTests/SPMIntegration/
+          name: Update git commit in targets that use SPM for dependencies
```

Comment:
`MagicWeather` and `MagicWeatherSwiftUI` have been recently updated to use the new identity v3 stuff, but to do that they need to point to a version of the SDK that has them. 

So I updated them to point to `main`, and in release versions, the current commit (this job takes care of updating said commit)